### DISCUSSION TITLE
fcitx5不能在wayland下的alacritty输入中文

### DISCUSSION THREAD

**Discussion by LzzKill** [fcitx/fcitx5#673] [Q&A] [answered]
我使用的是wayfire（wayland下很不错的窗口管理器）终端是alacritty，不仅仅在wayland下的alacritty不能使用fcitx5，neovide（一个neovim的client）也不能用fcitx5

**Comment by wengxt** [comment, marked as answer]
wayfire 没有支持 wayland 的输入法协议。 https://github.com/WayfireWM/wayfire/issues/1160

你唯一的选择是对这些程序用 xwayland
